src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -1614,21 +1610,81 @@ public void updateCounts( final MoleculeCounts moleculeCounts,
         return new Interval(readStart, readEnd);
     }
 
-    @VisibleForTesting static void updateCountsForPair( final ReadReport report1, final ReadReport report2 ) {
+    private static Interval calculateShortFragmentTrim( final GATKRead read, final Interval qualityTrim ) {
```

Comment:
I'd suggest give a short explanation on the difference between the two `calculate...Trim` functions.

Reply:
doc for trim methods added